### ISSUE TITLE
NTR: Blood pressure unstable (Autonomic)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7777]
New term request Blood pressure unstable (f267): Vasomotor instability, labile, hypertension, hypotension

**Comment by pnrobinson** [comment]
@MickeySegal
I do not think that the items in the list above are actually synonyms.
[Labile hypertension](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8673041/) = Labile hypertension is when a person's blood pressure frequently fluctuates between normal and high, changing more dramatically than normal.

[Vasomotor instability](https://www.sciencedirect.com/topics/neuroscience/vasomotor), also referred to as hot flashing or hot flushing, is a subjective sensation of heat associated with cutaneous vasodilation and a compensatory decrease in core body temperature that lasts about 4 minutes.

The HPO currently has [Episodic hypertension HP:0000875](https://hpo.jax.org/app/browse/term/HP:0000875) that is reserved for diseases such as Pheochromocytoma -- 
I think that a term for Labile hypertension might be good as a sibling of this term.

The HPO also has [Orthostatic hypotension HP:0001278](https://hpo.jax.org/app/browse/term/HP:0001278), which is more typically associated with autonomic dysfunction.

Please clarify

**Comment by MickeySegal** [comment]
I changed the explanatory terms to "labile, hypertension, hypotension; part of vasomotor instability".  The explanatory term are meant both for search and for explanation.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000321
- label: Labile Hypertension
- parents: [Hypertension (HP:0000822)](https://hpo.jax.org/app/browse/term/HP:0000822)
- pmid: 
- def: Frequent fluctuations of blood pressure between normal and high levels, changing more dramatically than normal.
- comment: It is normal for BP to fluctuate from moment to moment and from day to day. BP fluctuation is related to many factors such as physical activity, emotion, position, respiratory cycle, diet, salt intake, alcohol ingestion, sleep deprivation, and others. Even in otherwise normotensive individuals, BP fluctuation can be substantial during moments of physical or emotional stress or even without overt provocation.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7777
- synonyms:
